features/datalist.yml

```
@@ -1,5 +1,5 @@
 name: <datalist>
-description: The `<datalist>` element defines a set of recommended values for an `<input>` element. Browsers may show a dropdown menu of all options, or matching options as the user types.
+description: The `<datalist>` element sets a set of recommended values for an `<input>` element. Browsers may show a dropdown menu of all options, or matching options as the user types.
```

"sets a set" is no good. I think defines here is quite OK, and "provides" would also work.

---

"Defines" here is probably closer to the thing that the element does. That said, "sets" is still possible:

```suggestion
description: The `<datalist>` element sets recommended values for an `<input>` element. Browsers may show a dropdown menu of all options, or matching options as the user types.
```

Though I'm thinking the whole thing though is a bit messy. The second sentence is unclearly connected to the first and does a weird change in verb form. I'd maybe rewrite it like this:

```suggestion
description: The `<datalist>` element sets recommended values for an `<input>` element. Browsers may show a dropdown menu of all values or match values as the user types.
```